publish-share-code/intro.md

```diff
@@ -15,13 +15,34 @@ kernelspec:
 
 # Publish & share your code
 
-
 :::{toctree}
 :hidden:
 :caption: Share code
 :maxdepth: 2
 
 Intro <self>
-Zenodo <zenodo>
-GitHub <share-code-github>
+Share your Code <share-code>
+Cite your Code <cite-code>
 :::
+
+:::{toctree}
+:hidden:
+:caption: Activities 
+:maxdepth: 2
+
+
+Share your Code <activity-1-share-code>
+
+:::
+
+## Sharing and Citing Your Code
+
+Sharing your code using open, collaborative platforms is essential to open science because it adds transparency, encourages collaboration and reuse, and enables citability. 
```

Comment:
```suggestion
Sharing your code using open, collaborative platforms is essential to open science because it encourages collaboration and reuse, enables citability, and adds transparency needed for reproducibility. 
```